mods/ra/weapons/missiles.yaml

```diff
@@ -434,24 +304,33 @@ SubMissile:
 		ImpactSounds: splash9.aud
 		ValidImpactTypes: Water
 
+SubMissile:
+	Inherits: ^SubMissileDefault
+	Range: 16c0
+	-Projectile:
+	Projectile: Bullet
```

Comment:
Oh, never mind: https://github.com/OpenRA/OpenRA/pull/12633/commits/2bb7b188d1043de0993cb34d115a8e41b9881b40#r98308817 (This still looks bogus to me though. 😄)